app/controllers/catalog_controller.rb

```diff
@@ -150,6 +150,17 @@ def catalog_item_types
     }
   end
 
+  def generic_item_subtypes
+    {
+      "custom"        => "Custom",
+      "vm"            => "VM",
+      "instance"      => "Instance",
+      "load_balancer" => "Load Balancer",
+      "database"      => "Database",
```

Comment:
Do we want to call it "Database" or "Hosted Database" (hosted_database).  So, there is no confusion as to a VM with a DB server vs. Amazon RDS, or something like that?